.travis.yml

```diff
@@ -1,18 +1,34 @@
+dist: bionic
```

Comment:
Why not stretch?

Reply:
stretch was the 'testing' when the project started.